### DISCUSSION TITLE
AirBus Quadrant TCA ThrustMaster

### DISCUSSION THREAD

**Discussion by Naviador** [flybywiresim/aircraft#3831] [Show and tell]
Good Morning.
I did not know how to upload the file. To share with others. So I hope they don't scold me too much.
This is the configuration I have done for the Thrustmaster TCA Quadrant Airbus Edition
Since the axes did not move properly.
In the Sensitivity settings I will add a photo.

[Restart File ThrottleConfiguration.txt](https://github.com/flybywiresim/a32nx/files/6097263/Restart.File.ThrottleConfiguration.txt)
![Eliminar](https://user-images.githubusercontent.com/26739991/110242967-a8c0f680-7f58-11eb-94b0-1fcf2e3ce8a4.jpg)

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 1.5
DetentTakeOffGoAround = 1.00
DetentFlexMct = 0.50
DetentClimb = 0.30
DetentIdle = 0.0
DetentReverseIdle = -0.30
DetentReverseFull = -1.00

**Comment by JJKHub** [comment]
For those that want to setup their Quadrant, also check here:
https://github.com/flybywiresim/a32nx/tree/fbw/docs

**Comment by Naviador** [comment]
But here I can't write anything since I don't have permissions.

Отправлено из приложения "Почта<https://go.microsoft.com/fwlink/?LinkId=550986>" для Windows 10

От: JJKHub<mailto:notifications@github.com>
Отправлено: 7 марта 2021 г. в 16:21
Кому: flybywiresim/a32nx<mailto:a32nx@noreply.github.com>
Копия: Ivan St<mailto:stepi_uan_dia_d@hotmail.com>; Author<mailto:author@noreply.github.com>
Тема: Re: [flybywiresim/a32nx] AirBus Quadrant TCA ThrustMaster (#3831)


For those that want to setup their Quadrant, check here:
https://github.com/flybywiresim/a32nx/tree/fbw/docs

**Reply by JJKHub** [reply]
Yes I know and I appreciate your contribution. I just thought, that it might not be clear for everyone how to set/use these values, so I thought the link might help.

**Comment by ninoz** [comment]
What are people currently doing re the TCA setup and the mod?

I'm thinking of swaping back to development, loosing the throttleconfig.ini and just living with the non linear throttle.

**Comment by Naviador** [comment]
Good Morning.
The standard configuration is the one that comes by default in the simulator with the TCA Quadrant. They are misaligned.
For this reason, above has the configuration that I have adjusted. With mine it works perfectly.
The only thing I recommend is that you check your Throttle. Since the factory parameters may vary a bit.

To adjust the settings first the parameter.
DetentDeadZone = 0
This will allow you to see how it varies without stops at points such as TOGA, FLX or CL.
When you have already adjusted the percentages, the DetentDeadZone parameter must be given a comfortable value. I did it to make it look as realistic as possible and with the value 1.5 it is very realistic.
